pkg/annotation/aws.go

```diff
@@ -12,3 +12,14 @@ const AWSUpdateMaxBatchSize = "alpha.aws.giantswarm.io/update-max-batch-size"
 // The value must be in ISO 8601 duration format, e. g. "PT5M" for five minutes or "PT10S" for 10 seconds.
 // https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-attribute-updatepolicy.html#cfn-attributes-updatepolicy-rollingupdate-pausetime
 const AWSUpdatePauseTime = "alpha.aws.giantswarm.io/update-pause-time"
+
+// AWSMetadataV2 is the aws update annotation used for configuring
+// the state of token usage for your instance metadata requests.
```

Comment:
I still don't understand this sentence:

> AWSMetadataV2 is the aws update annotation used for configuring the state of token usage for your instance metadata requests.

Maybe just copy and adapt the entire description from upstream? Lines are for free.

```suggestion
// AWSMetadataV2 configures token usage for your instance metadata requests.
// If the value is 'optional', you can choose to retrieve instance metadata with or without a signed token
// header on your request. If you retrieve the IAM role credentials without a token, the version 1.0 role
// credentials are returned. If you retrieve the IAM role credentials using a valid signed token, the version
// 2.0 role credentials are returned.
// If the state is 'required', you must send a signed token header with any instance metadata retrieval
// requests. In this state, retrieving the IAM role credentials always returns the version 2.0 credentials; the
// version 1.0 credentials are not available.
// Default: 'optional'
```